hudi-common/src/main/java/org/apache/hudi/common/table/view/FileSystemViewManager.java

```diff
@@ -26,6 +26,7 @@
 import org.apache.hudi.common.function.SerializableSupplier;
 import org.apache.hudi.common.table.HoodieTableMetaClient;
 import org.apache.hudi.common.table.timeline.HoodieTimeline;
+import org.apache.hudi.common.util.Functions;
 import org.apache.hudi.common.util.Functions.Function2;
```

Comment:
import Function0, follow the existing style

Reply:
done